embeddingstore/embedding_store.cc

```diff
@@ -10,6 +10,10 @@
 
```

Comment:
use clang-format -style=google, on all relevant c++ (.cc and .h) files, make sure to not include BUILD or WORKSPACE or it'll break their formatting